packages/macro/test/fixtures/js-t-var/js-t-var.expected.js

```diff
@@ -1,20 +1,15 @@
 "use strict";
 
 var _core = require("@lingui/core");
-
 function scoped(foo) {
   if (foo) {
     var bar = 50;
-
-    /*i18n*/
-    _core.i18n._("This is bar {bar}", {
+    /*i18n*/_core.i18n._("This is bar {bar}", {
```

Comment:
The test fixtures in macro/test was updated because of newer babel/generator. Looks like newer version generates slightly different output in terms of whitespaces. This not affect logic anyhow, only formatting. 

So next you will see many changes related to this fact.